test/index.spec.js

```diff
@@ -228,25 +229,193 @@ describe('FlipMove', () => {
     });
   });
 
-  describe('duration prop runtime checking', () => {
-    let wrapper;
-    beforeEach(() => {
-      wrapper = shallow(<FlipMove />);
+  describe('prop runtime checking and conversion', () => {
+    let errorStub;
+    let warnStub;
+    const SFC = () => null;
+
+    before(() => {
+      errorStub = sinon.stub(console, 'error');
+      warnStub = sinon.stub(console, 'warn');
+    });
+    afterEach(() => {
+      errorStub.reset();
+      warnStub.reset();
     });
+    after(() => {
+      errorStub.restore();
+      warnStub.restore();
+    });
+
+    it('doesn\'t run checks in production environment', () => {
+      const env = process.env;
```

Comment:
Maybe this should be moved into the `before` block, and reset in the `afterEach`, similar to the console stubs? That way each test can provide whatever `NODE_ENV` they want without worrying